clients/wallet/src/horizon/tests.rs

```diff
@@ -170,7 +173,8 @@ async fn horizon_get_transaction_success() {
 #[tokio::test(flavor = "multi_thread")]
 async fn fetch_transactions_iter_success() {
 	let horizon_client = reqwest::Client::new();
-	let secret = secret_key_from_encoding(SECRET);
+	let is_public_network = false;
+	let secret = secret_key_from_encoding(&get_source_secret_key_from_env(is_public_network));
 	let fetcher = HorizonFetcher::new(horizon_client, secret.get_public().clone(), false);
```

Comment:
since we're introducing the variable `is_public_network`, we can replace `false` with that variable.

In Stellar, _public_ network is the **MAINNET**.

Reply:
Sure, I replaced in a few other places I see the boolean alone.